src/styles/sidebar/components/menu.scss

```diff
@@ -11,25 +12,16 @@
 // Toggle button that opens the menu.
 .menu__toggle {
   @include focus.outline-on-keyboard-focus;
```

Comment:
I don't believe you need to include `outline-on-keyboard-focus` directly anymore as you get that for free via any `button` derivative mixin.

Reply:
good catch!